web/cobrands/northamptonshire/assets.js

```diff
@@ -55,61 +43,74 @@ var layers = [
       "Restricted Visibility"
   ],
   "layer_name": "Tree",
-  "styleid": '5d481376fe2ad8059ce44ef2',
   "snap_threshold": 0
 },
 {
   "categories": [ "Safety Bollard - Damaged/Missing" ],
-  "layer_name": "Safety Bollard",
-  "styleid": "5d481446fe2ad8059ce44f02",
+  "layer_name": "Safety_Bollard"
 },
-];
-
-var prow_assets = [
 {
   "categories": [ "Bridge-Damaged/ Missing" ],
   "item_name": "bridge or right of way",
-  "layer_name": "BRIDGES",
-  "styleid": "5d48161ffe2ad809d85a7add"
+  "layer_name": "Bridges"
 },
 {
   "categories": [ "Gate - Damaged/ Missing" ],
   "item_name": "gate or right of way",
-  "layer_name": "GATE",
-  "styleid": "5d481906fe2ad8059ce450b4",
+  "layer_name": "Gates"
 },
 {
   "categories": [ "Stile-Damaged/Missing" ],
   "item_name": "stile or right of way",
-  "layer_name": "STILE",
-  "styleid": "5d481a05fe2ad8059ce45121",
+  "layer_name": "Stile"
 },
 {
   "categories": [ "Sign/Waymarking - Damaged/Missing" ],
   "item_name": "waymarking or right of way",
-  "layer_name": "WAYMARK POST",
-  "styleid": "5d481a4ffe2ad809d85a7b90&styleIds=5d481742fe2ad809d85a7b05"
+  "layer_name": "Waymarker"
+},
+{
+  "categories": [
+    "Damaged/Exposed Wiring / Vandalised",
+    "Lamp/Bulb Failure",
+    "Signal Failure",
+    "Signal Failure all out",
+    "Signal Stuck",
+    "Signal Head Failure",
+    "Request Timing Review",
+    "Damaged Control box",
+    "Signal Failure/Damaged - Toucan/Pelican"
+  ],
+  "item_name": "signal or crossing",
+  "layer_name": "Traffic_Signal_Junction"
 },
+{
+  "categories": [ "Pedestrian Barriers - Damaged / Missing" ],
+  "item_name": "pedestrian barrier",
+  "layer_name": "Pedestrian_Barrier"
+}
 ];
 
-var highway_layer = 'layers_highwayAssetsCustom_5d4806b0fe2ad809d85a774f';
-var prow_asset_layer = 'layers_pRoWAssets_5d48157cfe2ad809d85a7abc';
-var signal_asset_layer = 'layers_nETCOM_5d483dd7fe2ad809d85a8fab';
-
 // default options for northants assets include
 // a) checking for multiple assets in same location
 // b) preventing submission unless an asset is selected
-var northants_defaults = $.extend(true, {}, fixmystreet.alloyv2_defaults, {
-  class: OpenLayers.Layer.AlloyVectorAsset,
-  protocol_class: OpenLayers.Protocol.AlloyV2,
+var northants_defaults = {
   http_options: {
```

Comment:
As this is changing, and these are all standard WFS layers, we might as well use the slightly nicer code for WFS layers - so instead of http_options object, replace it entirely with wfs_url pointing at the same URL (none of params needed), and then in each assets.add, instead of http_options you only need wfs_feature: typename